airflow/docker.go

```diff
@@ -532,11 +532,12 @@ func Deploy(path, name, wsId string, prompt bool) error {
 		name = config.CFG.ProjectDeployment.GetProjectString()
 	}
 
+	if len(deployments) == 0 {
+		return errors.New(messages.HOUSTON_NO_DEPLOYMENTS_ERROR)
+	}
+
```

Comment:
this doesn't fix our current problem, but it fixes the client side error message to be correct in the case of houston not returning a list of deployments. i.e. if a user had a workspace with no deployments, then tried to deploy using a name, ```astro deploy this-deployment-was-not-created``` they would find the same issue. this is an informative error message instead.